modules/ob-understanding-the-status-of-a-build-run.adoc

```diff
@@ -0,0 +1,75 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-runs.adoc
+
+:_content-type: CONCEPT
+[id="ob-understanding-the-status-of-a-build-run_{context}"]
+= Understand the status of a build run
+
+A `BuildRun` resource can have different statuses during the image building process. The following table covers the different statuses of a build run:
+
+.Statuses of a build run
+[options="header"]
+|===
+
+| Status | Reason | Description
+
+| `Unknown` | `Pending` | The `BuildRun` resource waits for a pod in status `Pending`.
+
+| `Unknown` | `Running` | The `BuildRun` resource has been validated and started to perform its work.
+
+| `Unknown` | `BuildRunCanceled` | The user has requested to cancel the build run. This request triggers the build run controller to make a request for canceling the related task runs. Cancellation is still under process.
+
+| `True` | `Succeeded` | The pod for the `BuildRun` resource is created.
+
+| `False` | `Failed` | The `BuildRun` resource is failed in one of the steps.
+
+| `False` | `BuildRunTimeout` | The execution of the `BuildRun` resource is timed out.
+
+| `False` | `UnknownStrategyKind` | The strategy type defined in the `Kind` field is unknown. You can define these strategy types: `ClusterBuildStrategy` and `BuildStrategy`.
+
+| `False` | `ClusterBuildStrategyNotFound` | The referenced cluster-scoped strategy was not found in the cluster.
+
+| `False` | `BuildStrategyNotFound` | The referenced namespace-scoped strategy was not found in the cluster.
+
+| `False` | `SetOwnerReferenceFailed` | Setting the `ownerReferences` field from the `BuildRun` resource to the related `TaskRun` resource failed.
+
+| `False` | `TaskRunIsMissing` | The `TaskRun` resource related to the `BuildRun` resource was not found.
+
+| `False` | `TaskRunGenerationFailed` | The generation of a `TaskRun` specification has failed.
+
+| `False` | `MissingParameterValues` | You have not provided any value for some parameters that are defined in the build strategy without any default. You must provide the values for those parameters in the `Build` or the `BuildRun` CR.
+
+| `False` | `RestrictedParametersInUse` | A value for a system parameter was provided, which is not allowed.
+
+| `False` | `UndefinedParameter` | A value for a parameter was provided that is not defined in the build strategy.
+
+| `False` | `WrongParameterValueType` | A value was provided for a build strategy parameter with the wrong type. For example, the parameter is defined as an array or a string in the build strategy and you must provide a set of values or a direct value accordingly.
```

Comment:
Up to you! 
```suggestion
| `False` | `WrongParameterValueType` | A value was provided for a build strategy parameter with the wrong type. For example, if the parameter is defined as an incorrect type of an array or a string in the build strategy, you must provide a set of values or a direct value accordingly.
```

Reply:
Leaving the content as is because the suggested edits might change the intended meaning.